task/ko-oci-ta/0.1/ko-oci-ta.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: ko-oci-ta
+  annotations:
+    tekton.dev/categories: Image Build
+    tekton.dev/displayName: Build and upload container image using ko oci
+      trusted artifacts
+    tekton.dev/pipelines.minVersion: 0.17.0
+    tekton.dev/platforms: linux/amd64,linux/arm64
+    tekton.dev/tags: image-build
+  labels:
+    app.kubernetes.io/version: "0.1"
+spec:
+  description: This Task builds source into a container image using ko.
+  params:
+    - name: BUILD_ARGS
+      description: additional arguments to ko build
+      type: string
+      default: ""
+    - name: CACHI2_ARTIFACT
+      description: The Trusted Artifact URI pointing to the artifact with
+        the prefetched dependencies.
+      type: string
+      default: ""
+    - name: COMMIT_SHA
+      description: The image is built from this commit.
+      type: string
+      default: ""
+    - name: HERMETIC
+      description: Determines if build will be executed without network access.
+      type: string
+      default: "false"
+    - name: IMAGE
+      description: Reference of the image ko will produce.
+    - name: IMAGE_EXPIRES_AFTER
+      description: Delete image tag after specified time. Empty means to keep
+        the image tag. Time values could be something like 1h, 2d, 3w for
+        hours, days, and weeks, respectively.
+      type: string
+      default: ""
+    - name: IMPORT_PATH
+      description: import path of package main
+      default: .
+    - name: KO_DEFAULTBASEIMAGE
+      description: base image for ko build
+      default: ""
+    - name: KO_IMAGE
+      description: ko image to use for builds
+      default: quay.io/redhat-user-workloads/konflux-build-pipeli-tenant/ko-builder:127e70d57df2a6b75bf3a2bb233a6223fb6a8cab
```

Comment:
Use `quay.io/konflux-ci/ko-builder` instead?

Reply:
This image repository will depend on https://gitlab.cee.redhat.com/releng/konflux-release-data/-/merge_requests/10323, which I am awaiting approval on.  I don't really care where the image goes, as long as it goes somewhere where Konflux can get it.